Components/Image/Image.css

```diff
@@ -26,7 +26,7 @@
     /* left: 50%; */
     background-color: white;
     /* width: 100%; */
-    z-index: 9999;
+    z-index: 0;
```

Comment:
z-indexes are tricky to use, so if possible list the values as variables in root and then use the variables as when required.